website/docs/docs/cloud/about-cloud-develop.md

```diff
@@ -4,17 +4,27 @@ id: about-cloud-develop
 description: "Learn how to develop your dbt projects using dbt Cloud."
 sidebar_label: "About developing in dbt Cloud"
 pagination_next: "docs/cloud/cloud-cli-installation"
+hide_table_of_contents: true
 ---
 
-dbt Cloud offers a fast and reliable way to work on your dbt project. You can use it through an integrated development environment (IDE) in your browser or a dbt Cloud-powered command line interface (CLI):
+dbt Cloud offers a fast and reliable way to work on your dbt project. It runs dbt Core in a hosted (single or multi-tenant) environment with a browser-based integrated development environment (IDE) or a dbt Cloud-powered command line interface (CLI)[^1]. This allows you to seamlessly develop, test, and run dbt commands in the dbt Cloud IDE or through the dbt Cloud CLI.
 
-- **[dbt Cloud CLI](/docs/cloud/cloud-cli-installation)** &mdash; Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment.
+<div className="grid--3-col">
 
-  - Something to note is that the dbt Cloud CLI and the open-sourced dbt Core are both command line tools that let you run your dbt projects and use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure and integrates those builds with all [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
+<Card
+    title="dbt Cloud CLI"
+    body="Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment."
+    link="/docs/cloud/cloud-cli-installation"
+    icon="dbt-bit"/>
 
-- **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface (UI).
-
-This documentation section provides detailed instructions on setting up the dbt Cloud CLI and dbt Cloud IDE. To get started with dbt development, you'll need a [developer](/docs/cloud/manage-access/seats-and-users) account. For a more comprehensive setup guide, refer to our [quickstart guides](/quickstarts).
+  <Card
+    title="dbt Cloud IDE"
+    body="Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface (UI)."
```

Comment:
Suggest using "user interface" only because we don't use "UI" elsewhere.
```suggestion
    body="Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface."
```